programs/lz4io.c

```diff
@@ -412,13 +463,21 @@ static cRess_t LZ4IO_createCResources(void)
     ress.dstBuffer = malloc(ress.dstBufferSize);
     if (!ress.srcBuffer || !ress.dstBuffer) EXM_THROW(31, "Allocation error : not enough memory");
 
+    ress.cdict = LZ4IO_loadCDict();
+
     return ress;
 }
 
 static void LZ4IO_freeCResources(cRess_t ress)
 {
     free(ress.srcBuffer);
     free(ress.dstBuffer);
+
+    if (ress.cdict) {
+        LZ4F_freeCDict(ress.cdict);
```

Comment:
minor : for information, `LZ4F_freeCDict()` is compatible with free on `NULL`, so you could simplify this.
Btw, maybe this capability should be mentioned in `lz4frame_static.h`.